transcom/mymove
# B 21078 move history ppm cleanup

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A1010031&RoomContext=TeamRoom%3A848240&concept=TeamRoom)

## [INT PR](https://github.com/transcom/mymove/pull/13778)

## Summary

There were some cases when the Details column of the Move History tab was blank and not displaying useful information about the event shown.  This is a fix for the general flow of an PPM shipment through the Milmove application.

### How to test

1. Create or use an existing milmove customer and create a new move with an PPM shipment.
2. Take the shipment through the basic process and flow through Milmove.
NOTE:  The only areas I found that needed fixed and I addressed where as a customer, going back and updating something, (for example Closeout office) and going back and adding additional documents.  But please don't just test those only!
3. Check the Move History tab on the move and verify that there are no blanks in the Details column.
